src/backend/query_builder.rs

```diff
@@ -338,6 +343,24 @@ pub trait QueryBuilder: QuotedBuilder {
         .unwrap();
     }
 
+    /// Translate [`SelectDistinct`] into SQL statement.
+    fn prepare_select_lock(
+        &self,
+        select_lock: &LockType,
+        sql: &mut SqlWriter,
+        _collector: &mut dyn FnMut(Value),
+    ) {
+        write!(
+            sql,
+            "{}",
+            match select_lock {
```

Comment:
This trait impl things that are common across MySQL, Postgres and SQLite. Seemingly we cannot provide a default impl here, as things are different for Postgres and SQLite (which has no row based lock indeed).

Reply:
To be honest, I didn't check for sqlite before, and in fact now I find out it doesn't support row locking.
Postgres supports the same lock commands as MySQL, the docs are a little confusionary, all the lock types described are used internally.

I think we can leave this impl here and overwrite it in sqlite part

Reply:
How do you disable features in a single DB flavour?
ATM I've put a panic, but I don't feel good with it, at the same time writing code that does nothing IMHO isn't good too.

Reply:
I think 'do nothing' is the usual behavior

Reply:
Ok, I've removed the panic